Makefile

```diff
@@ -374,6 +383,45 @@ glance_deploy_cleanup: ## cleans up the service instance, Does not affect the op
 	rm -Rf ${OPERATOR_BASE_DIR}/glance-operator ${DEPLOY_DIR}
 	oc rsh -t mariadb-openstack mysql -u root --password=${PASSWORD} -e "drop database glance;" || true
 
+##@ OVN
+.PHONY: ovn_prep
+ovn_prep: export IMAGE=${OVN_IMG}
+ovn_prep: ## creates the files to install the operator using olm
+	$(eval $(call vars,$@,ovn))
+	bash scripts/gen-olm.sh
+
+.PHONY: ovn
+ovn: namespace ovn_prep ## installs the operator, also runs the prep step. Set OVN_IMG for custom image.
+	$(eval $(call vars,$@,ovn))
+	oc apply -f ${OPERATOR_DIR}
+
+.PHONY: ovn_cleanup
+ovn_cleanup: ## deletes the operator, but does not cleanup the service resources
+	$(eval $(call vars,$@,ovn))
+	bash scripts/operator-cleanup.sh
+	rm -Rf ${OPERATOR_DIR}
+
+.PHONY: ovn_deploy_prep
+ovn_deploy_prep: export KIND=OVN
```

Comment:
Thanks Andrew, yes right OVN doesn't exist as a Kind, i put it here just to get ovn deployed with current state with install_yamls but forgot to add a comment about it. And handle multiple CRDs later once ovn operator is more grown up.

Reply:
The problem this creates is that the `kustomize` logic won't be applied to the CRs being generated/deployed.  I just tried a `STORAGE_CLASS="" make ovn_deploy`, for instance, and the generated CRs still had `storageClass: local-storage`, so the `kustomize` failed to replace those fields with the empty-string.

Reply:
Yes right kustomize logic won't be applied at all, one way to handle it can be to update scripts/gen-service-kustomize.sh to include /spec/containerImage only when IMAGE != unused, wdyt?

Reply:
Running the `kustomize` isn't going to cause an explicit problem (as in an error), it's just that it won't actually do anything to the `OVNDBS` and `OVNNORTHD` CRs.  Checking for `IMAGE != unused` would not make a different either way.  The root issue is that the `kustomize` is looking for the CRs to have `Kind: ${KIND}`.  Since we currently only use that one env var (`KIND`), the pattern breaks down when trying to have `kustomize` target two or more CRs of different `Kind`s, because you cannot specify both `Kind`s in the single `KIND` env var.

This is where other operators instead have a top-level CRD that allows this pattern to work.  If we don't want to do the same thing for OVN, we'll have to either rethink how we do `kustomization` in `install_yamls` or perhaps just forgo doing anything of the sort for OVN deployment via this tool for now.

Reply:
I know how we can make this work for both CRs.  Change the `KIND` to `".*"`.  It appears that `kustomize` allows wildcards for field values.  Once that is in place, I think we can merge this.

Reply:
Thanks that looks good, Updated KIND to .* and not adding containerImage kustomize only when it's != unused

Reply:
Okay, and the check for `containerImage` being `"unused"` is actually something we should have had already, as we have other operators that use the `IMAGE=unused` pattern.  Thanks for adding that.